src/numpy.jl

```diff
@@ -170,7 +172,8 @@ const npy_typestrs = Dict( "b1"=>Bool,
                            "i8"=>Int64,     "u8"=>UInt64,
                            "f4"=>Float32,   "f8"=>Float64,
                            "c8"=>Complex64, "c16"=>Complex128,
-                           "O"=>PyPtr, "O$(div(Sys.WORD_SIZE,8))"=>PyPtr )
+                           "O"=>PyPtr, "O$(div(Sys.WORD_SIZE,8))"=>PyPtr,
+                           "f2"=>Float16)
```

Comment:
You should put this on the line with `f4` and `f8`.

(In general, when you make a patch, you want to make it blend in with the surrounding code, as if it were all written at one time by the same person.)